app/api/admin_statistics_api/users.py

```diff
@@ -62,7 +63,8 @@ def track_organizer_count(self, obj):
         return self.get_all_user_roles('track_organizer').count()
```

Comment:
I suspect this returns wrong count as well?

Reply:
@iamareebjamal actually not. That would work perfectly fine if we assign a user role of `track_organiser` 
The count would be returning 0 because currently there is no provision to assign roles to a user (at least it's not working properly, last I checked)